packages/vscode-extension/src/debug/constants.ts

```diff
@@ -86,3 +86,11 @@ Add any other context about the problem here.
 export const errorDetail = `
 **Error detail**
 `;
+
+export enum SideloadingHintStateKeys {
+  DoNotShowAgain = "sideloadingHint/doNotShowAgain",
+}
+export const sideloadingHintMessage =
+  "If any error occurs during local debug, you can refer to the FAQ page by clicking 'Open FAQ' button.";
```

Comment:
Why not move to Strings.json?

Reply:
Nice idea. Moved to `Strings.json` as well as `npmInstallFailedHintMessage` before. Thanks~